gui/dataTransfer.py

```diff
@@ -252,7 +252,7 @@ def run(self):
                                                    self.coll.path, newPath, scope="checksum")
                 self.updLabels.emit(len(onlyIrods), len(diff))
         except:
-            logging.exception("dataTransfer.py: Error in getDataState")
+            logging.exception('dataTransfer.py: Error in getDataState')
```

Comment:
Not sure if `logging.exception` is a deliberate choice here (over `logging.error`), but I think we'd better stick to `logging.error`. `logging.exception` generates additional lines of stacktrace that are not useful here.